packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -787,13 +797,52 @@ const DateTimePicker = ({
     setAbsoluteValue(newAbsolute);
   };
 
+  // Returns true if start date is after end date for absolute range
+  const invalidAbsoluteDate = (startTime, endTime, absoluteValues) => {
+    // If start and end date have been selected
+    if (absoluteValues.hasOwnProperty('start') && absoluteValues.hasOwnProperty('end')) {
+      // Include start time in start date
+      const timefrom = new Date(absoluteValues.start);
+      let temp = startTime.split(':');
+      timefrom.setHours(Number(temp[0]));
+      timefrom.setMinutes(Number(temp[1]));
+
+      // Include end time in end date
+      const timeto = new Date(absoluteValues.end);
+      temp = endTime.split(':');
+      timeto.setHours(Number(temp[0]));
+      timeto.setMinutes(Number(temp[1]));
+
+      return timefrom >= timeto;
+    }
+
+    // Return invalid date if start time and end date not selected
+    return true;
+  };
+
   // on change functions that trigger a absolute value update
   const onAbsoluteStartTimeChange = (pickerValue, evt, meta) => {
     setAbsoluteStartTimeInvalid(meta.invalid);
+    const invalidStartEndTime = invalidAbsoluteDate(
+      pickerValue,
+      absoluteValue.endTime,
+      absoluteValue
+    );
+    setAbsoluteStartTimeInvalid(meta.invalid || invalidStartEndTime);
+    setAbsoluteEndTimeInvalid(absoluteEndTimeInvalid && invalidStartEndTime);
```

Comment:
This will cause a bug if you:
1. Select an absolute date range 
2. Input an invalid number like `9999` in the "End time" input
3. Enter a valid number in the "Start time" input.
4. The "End time" input is still invalid but will no longer be shown as invalid because of the `absoluteEndTimeInvalid && invalidStartEndTime` where `invalidStartEndTime` is false

Reply:
Logic has been updated to account for this and to validate each TimrSpinner individually. Also, a test has been added to test this